app/gui/src/model/suggestion_database/mock.rs

```diff
@@ -0,0 +1,295 @@
+use crate::prelude::*;
+
+use crate::model::suggestion_database::entry;
+use crate::model::suggestion_database::entry::Argument;
+use crate::model::suggestion_database::Entry;
+use crate::model::SuggestionDatabase;
+
+use double_representation::name::QualifiedName;
+
+
+
+pub const DEFAULT_TYPE: &str = "Standard.Base.Any";
+
+#[derive(Debug)]
+pub struct Builder {
+    next_id:    entry::Id,
+    pub result: SuggestionDatabase,
+    in_module:  Option<QualifiedName>,
+    in_type:    Option<QualifiedName>,
+}
+
+impl Default for Builder {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl Builder {
+    pub fn new() -> Self {
+        Self {
+            next_id:   0,
+            result:    SuggestionDatabase::new_empty(),
```

Comment:
Why is this not default? Looks like the new empty state should be the default. Then this struct can derive Default